### DISCUSSION TITLE
Implement Adaptive Card Improved features

### DISCUSSION THREAD

**Discussion by SammyRasha** [cristianoag/microsoft-teams-apps-company-communicator#17] [Ideas]
Adaptive cards has some more features that would prove useful for CC -- 
1. Expanded view of images 
2. Full-width adaptive cards

![image](https://user-images.githubusercontent.com/10238857/168698057-e7b36738-dbcc-4ce6-af2f-7e47bbeb354a.png)
![image](https://user-images.githubusercontent.com/10238857/168698099-ecb8005b-020a-4970-a954-5bf4ab205a5b.png)

**Comment by cristianoag** [comment]
Expanded view for images is already part of the code.
